doc/source/whatsnew/v0.24.0.rst

```diff
@@ -372,6 +372,7 @@ Other Enhancements
 - :meth:`DataFrame.to_stata` and :class:`pandas.io.stata.StataWriter117` can write mixed sting columns to Stata strl format (:issue:`23633`)
 - :meth:`DataFrame.between_time` and :meth:`DataFrame.at_time` have gained the an ``axis`` parameter (:issue:`8839`)
 - :class:`IntervalIndex` has gained the :attr:`~IntervalIndex.is_overlapping` attribute to indicate if the ``IntervalIndex`` contains any overlapping intervals (:issue:`23309`)
+- :meth:`eval()` now supports use of `floor` and `ceil` in the expression
```

Comment:
Add the issue number here. This will also work in DataFrame.query, right?

Reply:
move this underneath this note:

``- Added log10 to the list of supported functions in :meth:`DataFrame.eval` (:issue:`24139`)``